vigiliae-christianae.csl

```diff
@@ -0,0 +1,790 @@
+            <date variable="issued">
+              <date-part name="year"/>
+            </date>
+          </else-if>
+          <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis article-journal" match="any">
+            <date variable="issued">
+              <date-part name="year"/>
+            </date>
+          </else-if>
+          <else>
+            <date variable="issued">
+              <date-part name="month" suffix=" "/>
+              <date-part name="day" suffix=", "/>
+              <date-part name="year"/>
+            </date>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="point-locators-subsequent">
+    <group>
+      <choose>
+        <if locator="page" match="none">
+          <choose>
+            <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+              <choose>
+                <if variable="volume">
+                  <group>
+                    <text term="volume" form="short" suffix=" "/>
+                    <number variable="volume" form="numeric"/>
+                    <label variable="locator" form="short" prefix=", " suffix=" "/>
+                  </group>
+                </if>
+                <else>
+                  <label variable="locator" form="short" suffix=" "/>
+                </else>
+              </choose>
+            </if>
+          </choose>
+        </if>
+        <else-if variable="volume">
+          <choose>
+            <if type="chapter paper-conference" match="any">
+              <choose>
+                <if position="subsequent">
+                  <group delimiter=" " suffix=", ">
+                    <text term="in"/>
+                    <text variable="container-title" font-style="italic"/>
+                    <text macro="container-contributors-short" prefix="(" suffix=")"/>
+                  </group>
+                </if>
+              </choose>
+              <number variable="volume" form="numeric" suffix=":"/>
+            </if>
+          </choose>
+        </else-if>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <macro name="point-locators">
+    <choose>
+      <if variable="locator" match="none">
+        <text macro="pages"/>
+      </if>
+      <else-if type="article-journal chapter" match="any">
+        <text macro="pages"/>
+        <group prefix=" " delimiter=" ">
+          <text term="at"/>
+          <text variable="locator"/>
+        </group>
+      </else-if>
+      <else>
+        <text macro="point-locators-subsequent" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal">
+        <text variable="page" prefix=" "/>
+      </if>
+      <else-if type="chapter paper-conference" match="any">
```

Comment:
this seems pointless -- looks like you could just do `<text variable="page" prefix=" "/>` for the entire macro, but if there are some item types (magazine and newspaper articles??) that you do want to exclude, you definitely only need one if and not an else-if with the same content.